charts/app-config/templates/kubescape.yaml

```diff
@@ -7,22 +7,21 @@ spec:
   destination:
     name: kubescape
     namespace: kubescape
-    server: 'https://kubernetes.default.svc' # change to your server
+    server: https://kubernetes.default.svc
   source:
-    path: charts/kubescape-operator
-    repoURL: 'https://github.com/kubescape/helm-charts'
-    targetRevision: HEAD
-    helm:
+    {{- include "app-config.helm-release"
```

Comment:
This only works if there's an entry in these files that you want renovate to keep up to date: https://github.com/alphagov/govuk-helm-charts/tree/main/charts/app-config/helm-versions

Reply:
ahh ok then no need for this - ok thank you